### ISSUE TITLE
The performance result

### ISSUE THREAD

**Issue by mingminzhen** [Lextal/pspnet-pytorch#1]
Do you have tried it on VOC or cityscape dataset?

**Comment by Lextal** [comment]
In progress

**Comment by AliBuildsAI** [comment]
any updates?

**Comment by jind11** [comment]
I have the same question, without large effective batch size (for batch normalization) like over 32, it is hard to reproduce the results in the original paper, and pytorch is not good at synchronizing different gpus

**Comment by ginobilinie** [comment]
It seems that the pyramid pooling module cannot really help improve the performance for me.

**Comment by xuwh15** [comment]
@Lextal Do you have codes for trying on cityscapes dataset?

**Comment by caijinyue** [comment]
any updates about the preformance？

**Comment by liberty-hit** [comment]
Any updates？
